### PR TITLE
WT-11598 Disable failing many-collection-test until server python changes fixed

### PR DESCRIPTION
**Pull request by ruby-oujo**

### CODE REVIEW

File: test/evergreen_develop.yml [hunk 1]
```diff
@@ -98,7 +98,8 @@ buildvariants:
     - name: many-dhandle-stress
     - name: ".wt2853-perf"
     - name: cppsuite-bounded-cursor-perf-stress
-    - name: many-collection-test
+    # FIXME-WT-11599 Disable until python changes are implemented.
+    # - name: many-collection-test
       distros: ubuntu2004-wt-large
```
Comment (mickgraham-mongodb) [inline test/evergreen_develop.yml:103]: This should be commented out as well.

Comment (mickgraham-mongodb) [reply test/evergreen_develop.yml:103]: The distros that is.

Comment (ruby-oujo) [reply test/evergreen_develop.yml]: done here [1c86b14](https://github.com/wiredtiger/wiredtiger/pull/9607/commits/1c86b1493581b1f71d32d9dba33f44fa076ad05d)

**Review by mickgraham-mongodb — APPROVED** [review]
LGTM

**Comment by ruby-oujo** [comment]
evergreen merge

**Pull request merged**